lib/src/widgets/custom_image_crop_widget.dart

```diff
@@ -188,45 +194,69 @@ class _CustomImageCropState extends State<CustomImageCrop> with CustomImageCropL
             Rect.fromCenter(
               center: Offset(width / 2, height / 2),
               width: cropWidth,
-              height: cropWidth,
+              height: cropHeight,
             ),
           );
     }
   }
 
+  Map<String, double> _getCropSize(double width, double height){
+    double cropWidth = width;
+    double cropHeight = height;
+    
+    if(widget.shape == CustomCropShape.Circle){
+      cropWidth = min(width, height) *  widget.cropPercentage;
+      cropHeight = cropWidth;
+    }
+
+    if(widget.shape == CustomCropShape.Square){
```

Comment:
I prefer a switch case, this way the linter will notify us if we missed a case (by adding new shapes for example)
Alternatively use an else if here so we don't do redundant checks

Reply:
I think it is inappropriate to use if else in switch, so I chose another way.